graph/meta.go

```diff
@@ -13,6 +13,7 @@ func NewMetadata() Metadata {
 
 // Metadata keys to be used instead of literal strings
 const (
+	Annotation      MetadataKey = "annotation"
```

Comment:
I have feeling using "annotation" naming will be confusing, "healthConfig" would help with the semantic.